src/views/intro/intro.scss

```diff
@@ -0,0 +1,30 @@
+.intro {
```

Comment:
Oh!  We do have a `intro.scss`.  Could we just move up the style flags to this file then?

Reply:
Skipping